src/service/repo/signedMessageRepo.ts

```diff
@@ -66,9 +68,10 @@ async function sendSignedMessage(
   channelId: string,
   message: string,
   channelType: number,
+  attachments: any,
   replyMessageId?: string
 ) {
-  let payload: SendPayloadType = {channelId, message, channelType};
+  let payload: SendPayloadType = {channelId, message, channelType, attachments};
```

Comment:
<img src="https://avatars.githubusercontent.com/in/347564?s=41" alt="Image description" width="20" height="20">   CodeRabbit

Again, the `attachments` parameter is of type `any`. Also, the payload creation can be simplified by including all properties in the initial declaration, instead of spreading them later.
```diff
-  attachments: any,
+  attachments: Attachment[],
...
-  let payload: SendPayloadType = {channelId, message, channelType, attachments};
-  if (replyMessageId) {
-    payload = {...payload, messageType: 'reply', replyMessageId};
-  }
+  let payload: SendPayloadType = {
+    channelId, 
+    message, 
+    channelType, 
+    attachments,
+    messageType: replyMessageId ? 'reply' : undefined,
+    replyMessageId
+  };
```



<!-- This is an auto-generated comment by OSS CodeRabbit -->